<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commodity</class>
 <widget class="QWidget" name="commodity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="newBtn_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/myImages/images/front_seal/add.png);</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteBtn_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/myImages/images/front_seal/delete.png);</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modifyBtn_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/myImages/images/front_seal/modify.png);</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>641</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
